### DISCUSSION TITLE
CSS for starter template

### DISCUSSION THREAD

**Discussion by san-sei** [thuliteio/doks#673] [Q&A] [answered]
@h-enk  I tried uploading `assets\scss\app.scss` by adding a line to import my custom CSS for the starter template. It worked on the dev environment but got an error at the production though. Do you know what might be the issue? or is there somewhere else that I need to reference the custom css file?

**Comment by h-enk** [comment, marked as answer]
Did you run `npm run test` locally and fix errors (if any)?

**Reply by san-sei** [reply]
@h-enk I am trying to host the build/public directory on GCP and it was working before but not anymore and getting the below error. Do you know what this dependency is and how can I remove this?

`Refused to apply style from 'https://doks.netlify.app/main.a786802e1bc087479c980d8e7d27c9fa9e9ab9776bfae0d624b059de9d94ceea017f4967e6b6e57754ef7cb98a7b17475b1d1364a967d16e172245425a2f35fd.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`

**Reply by h-enk** [reply]
Ah, that helps.

In `./layouts/partials/head/stylesheet.html` line 9, add ` | absURL`:

```html
  <link rel="stylesheet" href="{{ $secureCSS.Permalink | absURL }}" integrity="{{ $secureCSS.Data.Integrity }}" crossorigin="anonymous">
```

If that still doesn't work, please try with ` | relURL`.

**Reply by san-sei** [reply]
Just set the baseURL and everything works in production too as expected. Do you think it's still required to make the above change @h-enk

**Reply by h-enk** [reply]
No, setting `baseURL` correctly is enough
